src/layers/choropleth-layer/choropleth-layer.js

```diff
@@ -51,11 +51,11 @@ export default class ChoroplethLayer extends BaseLayer {
     attributeManager.addDynamic({
       // Primtive attributes
       indices: {size: 1, update: this.calculateIndices, isIndexed: true},
-      positions: {size: 3, update: this.calculatePositions},
-      colors: {size: 3, update: this.calculateColors},
+      positions: {size: 3, update: this.calculateInstancePositions},
+      colors: {size: 3, update: this.calculateInstanceColors},
```

Comment:
for choropleth layer, the positions and colors are not instanced.
(trying to pull these functions to the base-layer?)